src/performance/perfcollect/perfcollect

```diff
@@ -158,13 +158,27 @@ declare -a DotNETRuntime_TypeKeyword=(
 )
 
 declare -a DotNETRuntime_GCHeapDumpKeyword=(
+	DotNETRuntime:GCTriggered
```

Comment:
Thanks for making this change.  The arrays of events in this script were all generated by parsing the XML manifest - clretwall.man.  So, I'd like to keep the arrays as is unless you actually changed the manifest itself.  Instead of modifying the arrays, you can just call EnableLTTngEvents on multiple arrays - whichever correspond to what would be enabled on Windows.